eva-accession-import/src/test/resources/application.properties

```diff
@@ -4,3 +4,14 @@ dbsnp.datasource.username=SA
 dbsnp.datasource.password=
 dbsnp.datasource.schema=src/test/resources/dbsnp-mirror-schema.sql
 dbsnp.datasource.data=src/test/resources/dbsnp-mirror-data.sql
+
+parameters.fasta=src/test/resources/input-files/fasta/Gallus_gallus-5.0.test.fa
+parameters.assemblyReportUrl=file:src/test/resources/input-files/assembly-report/GCF_000002315.4_Gallus_gallus-5.0_assembly_report.txt
```

Comment:
Will the pipeline also support remote URLs, like the one to generate new accessions?

Reply:
yes. https://github.com/EBIvariation/eva-accession/blob/develop/eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/AssemblyReportReader.java#L56